### PR TITLE
Feature/js math and numbers

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
Add 2 exercises for js-math-and-numbers session.

### CODE REVIEW

**Comment by F-Kirchhoff** [comment]
Solution 1: https://codesandbox.io/s/collatz-solution-1-qrvj19?file=/js/index.js:605-634

**Comment by F-Kirchhoff** [comment]
Solution 1 (hyper lazer nyan): https://codesandbox.io/s/hyper-lazer-nyan-solution-1-d4cdbj?file=/js/index.js

File: sessions/js-math-and-numbers/hyper-lazer-nyan-13/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>HTML & CSS & JS</title>
```
Comment (marcbendixen) [inline sessions/js-math-and-numbers/hyper-lazer-nyan-13/index.html:6]:
```suggestion
		<title>Hyper Lazer Nyan</title>
```

File: sessions/js-math-and-numbers/hyper-lazer-nyan-13/js/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+console.clear();
+
+console.clear();
```
Comment (marcbendixen) [inline sessions/js-math-and-numbers/hyper-lazer-nyan-13/js/index.js:3]: Duplicate

File: sessions/js-math-and-numbers/hyper-lazer-nyan-13/package.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "static-html-css-js",
```
Comment (marcbendixen) [inline sessions/js-math-and-numbers/hyper-lazer-nyan-13/package.json:2]: I think the name needs to refer to the project!?

**Pull request merged**